components/atoms/button.html

```diff
@@ -1,21 +1,21 @@
-<div class="component" title="Button" version="1.1">
+<div class="component" title="Button" version="1.2">
   <includes>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
   </includes>
 
   <style>
-    .button {
```

Comment:
We actually need both `button` and `.button` here. You can use a comma to make a [selector list](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors#selector-list) so the rule applies to both.